R/bs-theme.R

```diff
@@ -280,7 +280,11 @@ bootstrap_bundle <- function(version) {
         system_file("components", "layout_column_wrap.scss", package = "bslib"),
         system_file("components", "sidebar.scss", package = "bslib"),
         system_file("components", "value_box.scss", package = "bslib")
-      ))
+      )),
+      # Enable CSS Grid powered Bootstrap grid
+      sass_layer(
+        defaults = list("enable-cssgrid" = TRUE)
+      )
```

Comment:
Should we set this as a default so users can later decide if they want to opt out?

```suggestion
      sass_layer(
        defaults = list("enable-cssgrid" = "true !default")
      )
```

Or IIUC we could give this layer a name, which would make it removable. In either case, we should probably document the opt-out method.

Reply:
Ah yea, it's probably not worth a bundle name, but it should definitely have a `!default` flag